### ISSUE TITLE
отсня́ться missing (-им-)

### ISSUE THREAD

**Issue by bzaar** [gramdict/zalizniak-2010#17]
A type-14 verb is said to be always accompanied by a present stem notation like (-н-), (-м-), (-им-) or something else, which is lacking in «отсня́ться св 14с/c''». This entry is not included in Zaliznjak (1980), but according to [Russian Wiktionary](https://ru.wiktionary.org/wiki/%D0%BE%D1%82%D1%81%D0%BD%D1%8F%D1%82%D1%8C%D1%81%D1%8F), (-им-) is expected here.

_Originally posted by @shenyileirob in https://github.com/gramdict/zalizniak-2010/issues/13#issuecomment-1369027447_

**Comment by bzaar** [comment]
I can confirm that this entry is missing (-им-) in my 2010 edition:

![image](https://user-images.githubusercontent.com/4028922/210269818-9a52138c-b2a0-48a8-8b28-8611b39969f7.png)

Indeed, most type 14 verbs have a (-xx-) type notation: https://gramdict.ru/search/*?index=14, including **отсня́ть** св 14c/c (-им-). Some exceptions exist, which do not have present/future forms: **объя́ть** св 14c, буд. нет, §17. But **отсня́ться** is obviously not one of them.

Bottom line, I agree that **отсня́ться** needs (-им-) and it's an error in the 2010 edition.
